Parsedown.php

```diff
@@ -84,6 +84,15 @@ function setSafeMode($safeMode)
 
     protected $safeMode;
 
+    function setHastagsEnabled($hashtagsEnabled)
```

Comment:
Apologies for not coming up with this earlier – could we call this `setStrictMode` and the respective property `strictMode`.

As suggested [here](https://github.com/erusev/parsedown/issues/265#issuecomment-70016695) I think that it would be good to group these opt-in spec improvements under one setter.